src/app/util/ember-compatibility-functions.cpp

```diff
@@ -248,6 +248,34 @@ bool ServerClusterCommandExists(const ConcreteCommandPath & aCommandPath)
 
 namespace {
 
+CHIP_ERROR ReadClusterDataVersion(const EndpointId & aEndpointId, const ClusterId & aClusterId, DataVersion & aDataVersion)
```

Comment:
Is there a reason to not pass a `const ConcreteClusterPath &` here?

Reply:
it seems there is no need to use ConcreteClusterPath. I have removed ConcreteClusterPath file.